tests/integration/components/content-test.js

```diff
@@ -0,0 +1,68 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { assertTooltipContent } from '../../helpers/ember-tooltips';
+
+moduleForComponent('tooltip-on-element', 'Integration | Option | content', {
+  integration: true,
+});
+
+test('assertTooltipContent correctly matches expected tootltip content', function(assert) {
+
+  assert.expect(2);
+
+  this.render(hbs`
```

Comment:
I think this makes the tests harder to read, but would be happy to change. My reasoning is that the surrounding div gives someone reading the test more context.

Again, happy to change, please just confirm.